app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,12 @@
+class PetApplicationsController < ApplicationController
```

Comment:
Excellent work creating this Joins controller! A very RESTful approach to setting up many to many joins records